Website/plugins/font-awesome/config.raku

```diff
@@ -10,5 +10,6 @@
 	:name<font-awesome>,
 	:render<fa-assets.raku>,
 	:template-raku<fa-template.raku>,
-	:version<0.4.1>,
+	:version<0.4.3>,
+	:information<jquery>,
```

Comment:
what does the :information key do?

Reply:
@dontlaugh basically it short-circuits the test for that field.
Most config fields are strings, but in order to allow for jquery and js files to be loaded in an order (eg., some js scripts expect variables to be declared by other js scripts, so they must be loaded by the browser later), the query gathering plugin allows for the field to be a sequence of arrays. 
I haven't figured out how to test this yet.
Is this not documented? Quite possibly not.
There is also another config field  `compare-ignore`. This one is for the `prepare-plugins` utility.
When a new plugin has been developed, or some have been changed, they are copied to directories that are named for the major version number. This is to allow for multiple versions of plugins to be available. 
This design means that a developer can keep using an older plugin, even if a new one that is not backwards compatible becomes available.
I worked on plugins for the GRAV CMS system. The problem is that every time GRAV was updated, the plugins had to be updated.

Reply:
I responded in detail in issue thread. It is a fudge factor when the tests would expect something else.